APMrover2/APMrover2.cpp

```diff
@@ -565,11 +569,12 @@ void Rover::update_navigation()
 
     case RTL:
         // no loitering around the wp with the rover, goes direct to the wp position
-        calc_lateral_acceleration();
-        calc_nav_steer();
         if (verify_RTL()) {
             SRV_Channels::set_output_scaled(SRV_Channel::k_throttle, g.throttle_min.get());
             set_mode(HOLD);
+        } else {
+            calc_lateral_acceleration();
+            calc_nav_steer();
```

Comment:
These changes and the one below have nothing to do guided velocity controller.  They are just a re-order.  Good idea but I would of preferred this to be in a separate commit. In fact if this was in a separate PR a quick test and I could of merged this straight in.  But as it stands it adds time and complexity to the review process.